### DISCUSSION TITLE
I'm using "ngx-translate" for translation. How can I use that in autoTips and form?

### DISCUSSION THREAD

**Discussion by MeAkib** [NG-ZORRO/ng-zorro-antd#8047] [General]
Hello,

I'm using "ngx-translate" package for translation. Want to use autoTips feature for validating form easily but couldn't. How can I use?
Here is autoTips feature:
https://ng.ant.design/components/form/en#components-form-demo-auto-tips
https://stackblitz.com/run?file=src%2Fapp%2Fapp.component.ts

Thanks

**Comment by ParsaArvanehPA** [comment]
Try using ngx-translate pipe. eg:
`[nzErrorTip]="'address.to.my.key' | translate"`
